BackToBasicsUIKit/BackToBasicsUIKit/Demo ViewControllers/Giannini/ProgrammaticSliderViewController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+import AVFoundation
+
+class ProgrammaticSliderViewController: UIViewController {
+    let slider = UISlider()
+    var imageView : UIImageView!
+    var songPlayer: AVAudioPlayer?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = UIColor.white
+        setupSlider()
+        setupBackground()
+        layoutConstraints()
+        DispatchQueue.global(qos: .userInitiated).async { self.playSong() }
+        }
```

Comment:
Why use GCD for this? Maybe you need to call `playSong()` on viewDidAppear?

Reply:
Removed GCD and used viewDidAppear.